packages/svelte-cloudinary/src/helpers/getCldOgImageUrl.ts

```diff
@@ -5,6 +5,27 @@ import type {
 	ImageOptions,
 } from '@cloudinary-util/url-loader';
 
+
+/**
+ * This can be used  to get a Cloudinary Open Graph image URL outside the component.
+ * 
+ * This helper provides a wide range of options for being able to easily generate social cards for Open Graph images.
+ * 
+ * Configuration for getCldOgImageUrl is the same as getCldImageUrl. 
```

Comment:
```suggestion
 * This can be used  to get a Cloudinary Open Graph image URL outside the component. This helper
 * provides a wide range of options for being able to easily generate social cards for Open Graph images.
 * Configuration for getCldOgImageUrl is the same as getCldImageUrl. It provides some sensible defaults
 * for open graph images, such as a width and height.
```